css/css-paint-api/dynamic-import.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<link rel="help" href="https://html.spec.whatwg.org/#hostimportmoduledynamically(referencingscriptormodule,-specifier,-promisecapability)">
+<link rel="match" href="dynamic-import-ref.html">
+<style>
+#output {
+    width: 100px;
+    height: 100px;
+    background-image: paint(rects);
+    background-color: blue;
+}
+</style>
+<script src="/common/reftest-wait.js"></script>
+<script src="/common/worklet-reftest.js"></script>
+<body>
+<div id="output"></div>
+
+<script id="code" type="text/worklet">
+registerPaint('rects', class {
+    async paint(ctx, geom) {
+        ctx.fillStyle = 'red';
+        try {
+          await import('./resources/imported-module.mjs');
+        } catch (e) {
+          if (e.constructor === TypeError) {
```

Comment:
Should we also check .name as testharness.js seems to do?

Reply:
Eh, the only way `.name` would be different would be if someone mucked with `TypeError.prototype.name`, which seems unlikely?